client/src/components/App.jsx

```diff
@@ -63,13 +63,12 @@ class App extends React.Component {
   }
 
 
-
   componentDidMount() {
     let initialFetchAttempts = 0;
     this.fetch({endpoint: 'products', params: {count: 1}},
       (data) => {
         console.log('[App] data recieved: ', data);
-        this.loadPage(data[0].id);
+        this.loadPage(28616);
```

Comment:
It's small enough to not worry about right now, but in the future, let's try to keep this argument as `data[0].id` for when we start actually employing page changes.